CMakeLists.txt

```diff
@@ -241,7 +242,8 @@ message(STATUS "SUSE 12: ${SSG_PRODUCT_SLE12}")
 message(STATUS "Ubuntu 14.04: ${SSG_PRODUCT_UBUNTU1404}")
 message(STATUS "Ubuntu 16.04: ${SSG_PRODUCT_UBUNTU1604}")
 message(STATUS "Ubuntu 18.04: ${SSG_PRODUCT_UBUNTU1804}")
-message(STATUS "WRLinux: ${SSG_PRODUCT_WRLINUX}")
+message(STATUS "WRLinux 8: ${SSG_PRODUCT_WRLINUX8}")
+message(STATUS "WRLinux 1019: ${SSG_PRODUCT_WRLINUX1019}")
```

Comment:
Needs to be added in `/build_product` script around line 74.

Reply:
Thank you. I will supply these to build_product script.

Reply:
@jan-cerny Nice catch!